inspirehep/modules/workflows/errors.py

```diff
@@ -26,5 +26,9 @@
 
 
 class DownloadError(WorkflowsError):
-
     """Error representing a failed download in a workflow."""
+
+
+class MissingHeadUUIDError(WorkflowsError):
+    """Error representing the missing field `head_uuid` inside the property
+    extra_data for a given workflow object."""
```

Comment:
But then people would complain that the automatic tool is rejecting their perfectly valid docstring, like in this case: https://github.com/inveniosoftware/troubleshooting/issues/9.

I prefer to educate people, so that they can make informed decisions on when to ignore rules that are anyway arbitrary.